src/utils/uploadUtils.ts

```diff
@@ -61,8 +61,8 @@ export function outputAndCopyUploadedFileUrls(parentUrl: string, fileUrls: strin
     }
 
     void vscode.window.showInformationMessage(
-        `Finished uploading ${fileUrls.length === 1 ? 'file' : `${fileUrls.length} files`}.`,
-        "Copy to Clipboard"
+        localize('outputAndCopyFinished', `Finished uploading ${fileUrls.length} ${fileUrls.length === 1 ? 'file' : 'files'}.`),
```

Comment:
Localize does formatting differently:
```suggestion
        localize('outputAndCopyFinished', 'Finished uploading {0} {1}.', fileUrls.length, fileUrls.length === 1 ? 'file' : 'files'),
```

Reply:
Oof, thank you for pointing this out to me.  I think I did it right once or twice before and then proceeded to not use it correctly every other time after... 😧

I did notice that there were other places that I had not touched that were not always using it where they (probably) should be... I might look into fixing those in the near future.

Reply:
Well, technically the interpolated string will work, but, it would not really permit for real localization, if we ever actually do that.